OtripleS.Web.Api/Brokers/DateTimes/DateTimeBroker.cs

```diff
@@ -5,5 +5,14 @@ namespace OtripleS.Web.Api.Brokers.DateTimes
     public class DateTimeBroker : IDateTimeBroker
     {
         public DateTimeOffset GetCurrentDateTime() => DateTimeOffset.UtcNow;
+
+        public DateTimeOffset GetRandomDate()
```

Comment:
I couldn't find the `DateRange` namespace so I created this function instead

Reply:
Try this:
```csharp
        private static DateTimeOffset GetRandomDateTime() =>
            new DateTimeRange(earliestDate: new DateTime()).GetValue();
```